File: Exercism Codes/C#/LuciansLusciousLasagna.cs
class Lasagna
{
    public int ExpectedMinutesInOven(){
        return 40;
    }
    public int RemainingMinutesInOven(int rem){
        return ExpectedMinutesInOven() - rem;
    }
    public int PreparationTimeInMinutes(int layer){
        return layer*2;
    }
    public int ElapsedTimeInMinutes(int layer, int rem){
        return rem + PreparationTimeInMinutes(layer);
    }
}

File: Exercism Codes/F#/LuciansLusciousLasagna.fs
module LuciansLusciousLasagna

let expectedMinutesInOven = 40

let remainingMinutesInOven x : int = expectedMinutesInOven - x

let preparationTimeInMinutes x : int = x * 2

let elapsedTimeInMinutes x y : int = y + preparationTimeInMinutes x